compiler/pkg/generator/template/client.go.tmpl

```diff
@@ -545,6 +568,21 @@ func (group *{{$node.GroupTypeName}}) Create{{$node.BaseNodeName}}ByName(ctx con
 		}
 	}
 	{{ else }}
+	if exists {
+		var parent *{{$node.Parent.SimpleGroupTypeName}}{{$node.Parent.BaseNodeName}}
+		var err2 error
```

Comment:
Why do we need a seperate declaration of these variables?

Reply:
Right @kacpersaw , we don't necessarily need to declare these variables separately.
I have updated the code PTAL again.